indigo_app/templates/indigo_api/task_list.html

```diff
@@ -9,50 +9,14 @@
     <a href="{% url 'create_task' place=place.place_code %}" class="btn btn-success">New task</a>
   </div>
 
-  {% if tasks %}
-    <div class="card">
-      <div class="list-group list-group-flush">
-        {% for task in tasks %}
-          <div class="list-group-item p-3 d-flex align-items-center">
-            <div class="col text-center">
-              <span class="badge task-badge task-badge-{{ task.state }}" title="{{ task.state }}"><i class="fas fa-fw task-icon-{{ task.state }}"></i></span><br>
-              <span class="text-muted">#{{ task.pk }}</span>
-            </div>
-
-            <div class="col flex-fill">
-              <div>
-                <a href="{{ task.pk }}/">{{ task.title }}</a>
-                {% for label in task.labels.all %}
-                  <span class="badge badge-secondary" title="{{ label.description }}">{{ label.title }}</span>
-                {% endfor %}
-
-                {% if task.document %}
-                  <div>
-                    <a href="{% url 'document' doc_id=task.document.id %}" class="text-muted">{{ task.document.title }} @ {{ task.document.expression_date|date:'Y-m-d' }} · {{ task.document.language }}</a><br>
-                    <span class="text-muted">{{ task.document.work.frbr_uri }}</span>
-                  </div>
-                {% elif task.work %}
-                  <div>
-                    <a href="{% url 'work' frbr_uri=task.work.frbr_uri %}" class="text-muted">{{ task.work.title }}</a><br>
-                    <span class="text-muted">{{ task.work.frbr_uri }}</span>
-                  </div>
-                {% endif %}
-              </div>
-              <div class="text-muted">
-                <small>
-                  Created <span class="time-ago" data-timestamp="{{ task.created_at|date:'c' }}">{{ task.created_at|naturaltime }}</span>
-                  {% if task.created_by_user %}
-                  by {% user_profile task.created_by_user %}
-                  {% endif %}
-                </small>
-              </div>
-            </div>
-          </div>
-        {% endfor %}
-      </div>
-    </div>
+  {% if frbr_uri %}
```

Comment:
I don't think you need to do this -- the list of tasks is already passed into the template as context. eg. for the task listing page, it'll be paginated. You can rely on the `tasks` already in the context.

We did this for the old workflows because we weren't able to override the view logic to add tasks to the context.